snap/snapcraft.yaml

```diff
@@ -308,6 +318,9 @@ parts:
       # Remove scripts that we'll be replacing with our own
       - -support-files/mysql.server
       - -COPYING
+    stage-packages:
+      # This seems to be needed for core18
+      - libaio1
```

Comment:
When specifying `base`, snapcraft no longer slurps up libraries from the build-packages to put into the snap. As a result, stage-packages are required. Rather than staging the entire deb though, can you stage only the libraries that are required (using the stage keyword, for example)?

Reply:
OK, I will try that, but I don't get what's the benefit of doing so, at least in this particular case, as the library is only 30KB (https://packages.ubuntu.com/bionic/libaio1) and it contains only a few files https://packages.ubuntu.com/bionic/amd64/libaio1/filelist ...what am I missing?

Reply:
Indeed, this is general guidance. We'll end up with a lot of cruft we didn't have before if we just stage entire packages, and in some cases it will be a lot worse than 30KB. If it gets insane remember you can make stage-packages standalone parts as well, using the `nil` plugin.

Reply:
It ended up being about 500K altogether. Not enough to justify the trouble, I think. Scratch my advice for now.